sdk/go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/google/go-cmp v0.5.9
 	github.com/stretchr/testify v1.8.1
+	golang.org/x/sys v0.4.0
```

Comment:
I think for sake of this PR you can revert the `go.mod` and `go.sum` changes.
Reference: https://github.com/open-telemetry/opentelemetry-go/actions/runs/4240270160/jobs/7369532517

Reply:
What if i run go mod tidy instead? wouldn't it be better than reverting the commit ? Let me know what you think

Reply:
I think it will be easier (mainly for the reviewers) if you revert and make a separate issue and PR for bumping `golang.org/x/sys v0.4.0`.

Otherwise you should update the PR description and write that (and why) are you bumping `golang.org/x/sys`.

Reply:
@pellared i have reverted go mod and go sum changes. Thanks for the review